app/models/voxelytics/VoxelyticsService.scala

```diff
@@ -77,14 +77,16 @@ object ChunkCounts {
 
 case class WorkflowListingRunEntry(id: ObjectId,
                                    name: String,
-                                   username: String,
+                                   hostusername: String,
                                    hostname: String,
                                    voxelyticsVersion: String,
```

Comment:
upon first sight it seems to me that the naming convention is mixed here (snake case vs. camel case vs. all small letters). Would this be an opportunity to unify the properties or is it on purpose?

Reply:
Yes, feel free to camelCase this stuff!

Also, hostusername can probably be thrown out, as the frontend does not display it anymore?

Reply:
alright, I'll do that! actually the user name on the host is displayed if the wk user couldnt be found, so it is still used. is it okay if I keep it in?

Reply:
Ah now I see the left join. Ok fair, if we want to support the rare case of a deleted user. However, in this case the type in scala needs to be Option[String]. And please test locally once that the query still behaves correctly if the user is missing/deleted.

But yes, then let’s keep the hostusername in as a fallback.

Reply:
local testing after deleting the user from the database went well :) 
![image](https://github.com/scalableminds/webknossos/assets/57364771/26be86d9-9b60-415f-b401-9da1363e4156)